addons/medical_damage/stringtable.xml

```diff
@@ -687,5 +687,11 @@
             <Polish>Ustawia próg obrażeń jakie może otrzymać jednostka przed utratą przytomności. (0 jako ustawienie domyślne misji)</Polish>
             <Turkish>Bilinçsiz duruma geçmeden önce bir birimin alabileceği hasar miktarını ayarlar. (Görev varsayılanı için 0)</Turkish>
         </Key>
+        <Key ID="STR_ACE_Medical_Damage_painUnconsciousChance_DisplayName">
```

Comment:
Capitalize first letter to match other stringtable keys here and for description: `STR_ACE_Medical_Damage_PainUnconsciousChance_DisplayName`.